chucks-pick3-mobile/src/app/services/state.service.ts

```diff
@@ -5,14 +5,22 @@ import { NavigationEnum } from '../models/navigate.enum';
 @Injectable({
   providedIn: 'root',
 })
-export class StateDrawDateService {
+export class StateService {
   private closedDatesArray = {
```

Comment:
Lets make this an actual array via `private closedDayArray = [{state: 'TX', days: ['Sunday']}]` 

Really and truly we should probably be using the Map from collectionsjs.com.   For example, 
`
const closedDayMap = new Map();

closedDayMap.set('TX', ['Sunday']);
`